scripts/override-start.js

```diff
@@ -24,8 +24,28 @@ const overrideWebpackConfig = ( config, devServer ) => {
 	// Replace the react-dev-utils webpackHotDevClient with a version patched to
 	// correctly detect the dev server host & port for socket requests.
 	const hotClient = require.resolve( 'react-dev-utils/webpackHotDevClient' );
-	const hotClientIndex = config.entry.indexOf( hotClient );
-	config.entry.splice(hotClientIndex, 1, require.resolve( '../overrides/webpackHotDevClient' ) );
+
+	if ( config.entry instanceof Array ) {
+		const hotClientIndex = config.entry.indexOf( hotClient );
+		config.entry.splice( hotClientIndex, 1, require.resolve( '../overrides/webpackHotDevClient' ) );
```

Comment:
Why not check for `>= 0` here? Like below in the `forEach`...

Reply:
Copy paste from the original code, but makes sense while we're making things robust